bedrock/firefox/urls.py

```diff
@@ -36,8 +36,6 @@
         ),
     ),
     path("firefox/all/", views.firefox_all, name="firefox.all"),
-    page("firefox/browsers/", "firefox/browsers/index.html", ftl_files=["firefox/browsers"]),
-    page("firefox/products/", "firefox/products/index.html", ftl_files=["firefox/products"]),
```

Comment:
Note for reviewer: this should have been removed in https://github.com/mozilla/bedrock/issues/14221 but got added back accidentally in a rebase.